src/Sets/Tetrahedron.jl

```diff
@@ -0,0 +1,111 @@
+export Tetrahedron
+
+"""
+    Tetrahedron{N, VN<:AbstractVector{N}, VT<:AbstractVector{VN}} <: AbstractPolytope{N}
+
+Type that represents a tetrahedron in vertex representation.
+
+### Fields
+
+- `vertices` -- list of vertices
+
+### Examples
+
+A tetrahedron can be constructed by passing the list of vertices.
+The following builds the tetrahedron with edge length 2 from the [wikipedia page Tetrahedron](https://en.wikipedia.org/wiki/Tetrahedron): 
+
+```jldoctest
+julia> vertices = [[1, 0, -1/sqrt(2)], [-1, 0, -1/sqrt(2)], [0, 1, 1/sqrt(2)], [0, -1, 1/sqrt(2)]];
+
+julia> T = Tetrahedron(vertices); T isa Tetrahedron
+true
+
+julia> zeros(3) ∈ T
+true
+
+julia> dim(T)
+3
+
+julia> σ(ones(3), T)
+3-element Vector{Float64}:
+ 0.0
+ 1.0
+ 0.7071067811865475
+```
+"""
+struct Tetrahedron{N,VN<:AbstractVector{N},VT<:AbstractVector{VN}} <: AbstractPolytope{N}
+    vertices::VT
```

Comment:
I added a constructor with a quick sanity check.

```suggestion
    vertices::VT

    function Tetrahedron(vertices::AbstractVector)
        @assert length(vertices) == 4 "a tetrahedron requires four vertices"
        return new(vertices)
    end
```

Maybe we should also check for linear independence. The check can be made optional, and in general we can easily deactivate `@assert`s for benchmarks.

Reply:
i'd leave it general for now, but can add a check that it is actually a tetrahedron  later.